### ISSUE TITLE
wordpress user homedir can't be in other directory than /home

### ISSUE THREAD

**Issue by Spockie** [inmotionhosting/ansible-role-wordpress#6]
in my private repo I have amended it this way:

```
diff --git a/playbooks/roles/inmotionhosting.wordpress/defaults/main.yml b/playbooks/roles/inmotionhosting.wordpress/defaults/main.yml
index 89986e9..f347f6b 100644
--- a/playbooks/roles/inmotionhosting.wordpress/defaults/main.yml
+++ b/playbooks/roles/inmotionhosting.wordpress/defaults/main.yml
@@ -15,11 +15,12 @@ site_pass: "example_password"
 
 # The system user that the WordPress site will belong to.
 system_user: "wordpress"
+system_user_home_path: "/home/{{ system_user }}"
 
 # The name of the folder in the above user's home directory that
 # WordPress will be installed to.
 wp_system_folder: "doc_root"
-wp_system_path: "/home/{{ system_user }}/{{ wp_system_folder }}"
+wp_system_path: "{{ system_user_home_path }}/{{ wp_system_folder }}"
 
 # These variables control the database name, user, and the user's
 # password that WordPress will use in the wp-config.php file.
diff --git a/playbooks/roles/inmotionhosting.wordpress/tasks/main.yml b/playbooks/roles/inmotionhosting.wordpress/tasks/main.yml
index bada2f3..231bf5f 100644
--- a/playbooks/roles/inmotionhosting.wordpress/tasks/main.yml
+++ b/playbooks/roles/inmotionhosting.wordpress/tasks/main.yml
@@ -14,10 +14,11 @@
   user:
     name: "{{ system_user }}"
     group: "{{ system_user }}"
+    home: "{{system_user_home_path}}"
 
 - name: (WordPress) Ensure system user home has execute permissions
   file:
-    path: "/home/{{ system_user }}"
+    path: "{{system_user_home_path}}"
     mode: "0711"
 
 #
diff --git a/playbooks/roles/inmotionhosting.wordpress/templates/etc/httpd/conf.d/site.conf.j2 b/playbooks/roles/inmotionhosting.wordpress/templates/etc/httpd/conf.d/site.conf.j2
index c3292fa..7ea4804 100644
--- a/playbooks/roles/inmotionhosting.wordpress/templates/etc/httpd/conf.d/site.conf.j2
+++ b/playbooks/roles/inmotionhosting.wordpress/templates/etc/httpd/conf.d/site.conf.j2
@@ -20,10 +20,10 @@
         SetHandler proxy:fcgi://{{ system_user }}
     </FilesMatch>
 
-    DocumentRoot /home/{{ system_user }}/{{ wp_system_folder }}/
+    DocumentRoot {{ wp_system_path }}
     DirectoryIndex index.html index.php
 
-    <Directory /home/{{ system_user }}/{{ wp_system_folder }}/>
+    <Directory {{ wp_system_path }}>
         Options FollowSymLinks
         AllowOverride All
         Require all granted
@@ -53,10 +53,10 @@
         SetHandler proxy:fcgi://{{ system_user }}
     </FilesMatch>
 
-    DocumentRoot /home/{{ system_user }}/{{ wp_system_folder }}
+    DocumentRoot {{ wp_system_path }}
     DirectoryIndex index.html index.php
 
-    <Directory /home/{{ system_user }}/{{ wp_system_folder }}>
+    <Directory {{ wp_system_path }}>
         Options FollowSymLinks
         AllowOverride All
         Require all granted
```

**Comment by Respheal** [comment]
Thanks for the report! This was intentional as our goal was to allow these roles and playbooks to be used in servers with multiple users. However, we've taken this report into consideration and may, in the future, amend the playbooks/roles to allow installation outside of /home. I'm leaving this issue open for now as this is still under consideration.

**Comment by jcmdln** [comment]
I think the best path forward would be to simply use `wp_system_path` to adjust the path and update our templates to use this variable rather than redefining the full path yet again. I'll get this change in momentarily.

**Comment by jcmdln** [comment]
Implemented in a33f5a8d, Thanks for catching this @Spockie!
